### ISSUE TITLE
No colors output for non TTY environment like for example GitLab CI

### ISSUE THREAD

**Issue by tymonx** [mgechev/revive#418]
**Describe the bug**
No colors output for non TTY environment like for example GitLab CI. The [kyoh86/richgo](https://github.com/kyoh86/richgo) tool have the same issue but it provides the `RICHGO_FORCE_COLOR` environment variable to force colors output to non TTY. The `revive` tool should have something similar.

**To Reproduce**
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file as a GitLab CI job using the Docker image `golang:1.14.2-alpine3.11`:

```shell
revive -formatter friendly ${GO_LINT_PATHS:-./...}
```

```toml
# config file $HOME/revive.toml
ignoreGeneratedHeader = false
severity = "warning"
confidence = 0.8
errorCode = 1
warningCode = 2

[rule.blank-imports]
[rule.context-as-argument]
[rule.context-keys-type]
[rule.dot-imports]
[rule.error-return]
[rule.error-strings]
[rule.error-naming]
[rule.exported]
[rule.if-return]
[rule.increment-decrement]
[rule.var-naming]
[rule.var-declaration]
[rule.package-comments]
[rule.range]
[rule.receiver-naming]
[rule.time-naming]
[rule.unexported-return]
[rule.indent-error-flow]
[rule.errorf]
[rule.empty-block]
[rule.superfluous-else]
[rule.unused-parameter]
[rule.unreachable-code]
[rule.redefines-builtin-id]
```

**Expected behavior**
Colors man. Colors.

**Logs**
GitLab CI job: https://gitlab.com/tymonx/gitlab-ci/-/jobs/552912963

![image](https://user-images.githubusercontent.com/8367378/81962023-6c13bf80-9613-11ea-9476-5b3a5538d271.png)

**Desktop (please complete the following information):**
 - OS: running the Docker image `golang:1.14.2-alpine3.11` under GitLab CI
 - Version of Go: 1.14.2

**Additional context**
None.

**Comment by tymonx** [comment]
I have added support for the `REVIVE_FORCE_COLOR` environment variable in this PR #420. It will force colorizing for no TTY environments. It is similar to the `richgo` project that use `RICHGO_FORCE_COLOR=1` for that.

Tested:
```shell
TERM=dumb REVIVE_FORCE_COLOR=1 revive -formatter friendly ./...
``` 

The `TERM` environment variable is handled internally by the `fatih/color` package. It disables colorizing when `TERM=dumb` by setting the global `color.NoColor` boolean variable to `true`. This emulates the no TTY use case.

Setting the `REVIVE_FORCE_COLOR=1` environment variable will overwrite this behavior.

**Comment by tymonx** [comment]
@mgechev can you look into that? It is OK?
